src/toil/provisioners/clusterScaler.py

```diff
@@ -416,6 +433,91 @@ def _round(self, number: float) -> int:
             # If we get here, something has gone wrong.
             raise RuntimeError(f"Could not round {number}")
 
+    def _reserve_overhead(self, full_node: Shape) -> Shape:
```

Comment:
```
None of this is user-configurable.
```

Would we ever want to disable / overwrite the overhead? Some tests are not passing because the limit is set to a very small value, but I don't know if that will happen realistically.

Reply:
I think users might run into situations where we thing a job won't fit when it really will fit. It might make sense to give people a way to turn this back off, since it's possible that they already know a workflow works and we'd be guessing it wouldn't work. But I'm having trouble thinking of a good way to let people actually tune this.